### DISCUSSION TITLE
Welcome to magixcms-3 Discussions!

### DISCUSSION THREAD

**Discussion by gtraxx** [magix-cms/magixcms-3#136] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by azoturie** [comment]
Hi, after reading carefully the install docs, you spoke about an app/config folder with a necessary chmod 777, but this folder is not present in the archive of the version 3.5
Is this the reason why I receive a #500 error when I connect to the xxxx/install page ?
I'm working on a distant server with php 7.2

thanks

**Reply by gtraxx** [reply]
Hi, this is the app/init folder
Regarding the 500 error, do you have the apache logs ?
Is it installed in a folder or in the root of your hosting ?

Best regards
Aurelien

**Comment by azoturie** [comment]
Aurelien,

 

 

Merci.

 

Entretemps l’installation s’est bien déroulée sans modifier les droits des fichiers (qui se mettaient déjà à 755).

J’apprécie l’approche de ce CMS.

 

En ce qui concerne l’erreur 500, c’était curieusement sur de NAS locaux que je l’ai rencontrée, mais je vais investiguer du côté de logs, et sur un serveur externe en sous-domaine avec un folder racine propre à ce nom de sous-domaine, donc pas le root du domaine principal.

 

Bien à vous,

 

André Moreau

 

 

 

 

De : Gérits Aurélien ***@***.***> 
Envoyé : jeudi 31 mars 2022 16:36
À : magix-cms/magixcms-3 ***@***.***>
Cc : azoturie ***@***.***>; Comment ***@***.***>
Objet : Re: [magix-cms/magixcms-3] Welcome to magixcms-3 Discussions! (Discussion #136)

 

Hi, this is the app/init folder
Regarding the 500 error, do you have the apache logs ?
Is it installed in a folder or in the root of your hosting ?

Best regards
Aurelien

—
Reply to this email directly, view it on GitHub <https://github.com/magix-cms/magixcms-3/discussions/136#discussioncomment-2478846> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AD3M36DIFVFOFSC36BH63UDVCWZ4PANCNFSM4UTCUNWQ> .
You are receiving this because you commented.  <https://github.com/notifications/beacon/AD3M36AYV7TYQQIR6TECUDLVCWZ4PA5CNFSM4UTCUNW2YY3PNVWWK3TUL52HS4DFWFCGS43DOVZXG2LPNZBW63LNMVXHJKTDN5WW2ZLOORPWSZGOAAS5F7Q.gif> Message ID: ***@***.*** ***@***.***> >



-- 
L'absence de virus dans ce courrier électronique a été vérifiée par le logiciel antivirus Avast.
https://www.avast.com/antivirus
